scripts/base/init-bare.bro

```diff
@@ -3710,11 +3710,36 @@ export {
 	## external harness and shouldn't output anything to the console.
 	const errors_to_stderr = T &redef;
 }
+
+module Fanout;
+
+type Method: enum {
```

Comment:
Agreed, there is no need for the option. I'll settle on hash for the patch update. Having the defrag option might also be unnecessary.